.github/workflows/test.yml

```diff
@@ -5,6 +5,9 @@ on: [push, pull_request]
 
 env:
   CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+  SONAR_PROJECT: react-native-base
```

Comment:
Should we need the SONAR_PROJECT key here?

Reply:
I guess it is not strictly necessary, we could just reference the project name directly in the sonar execution, it is just cleaner to modify later

Reply:
@sebalopez you could add this as another secret VAR, so to replay this example we will just need to change secrets, instead of secrets and config code.